### ISSUE TITLE
Fastpages style blog page

### ISSUE THREAD

**Issue by Isaac-Flath** [fastai/nbprocess#127]
My understanding is that nbprocess is going to be nbdev2, but that it will also replace fastpages.  This issue is relating to the fastpages/blogging style projects. 

Is there a built in way to create the main page of the blog that lists blog posts?  

For example:
+ Fastpages creates this page automatically from the notebooks front matter:  https://isaac-flath.github.io/fastblog/
+ Quarto default blog creates this page automatically from notebooks front matter:  https://isaac-flath.github.io/APL-Exploration/

I think something like this is important for a blog so that people can see an order list of posts, with dates and description of post content.

**Comment by hamelsmu** [comment]
Yeah we will have to write documentation on how to do this.  Its pretty straightforward but you have to change several settings, and maybe change something in nbprocess

https://quarto.org/docs/websites/website-blog.html
